src/redist/redist.csproj

```diff
@@ -84,6 +84,7 @@
       <RoslynDeps Include="$(RoslynDirectory)/bincore/csc.deps.json" />
       <RoslynDeps Include="$(RoslynDirectory)/bincore/vbc.deps.json" />
       <RoslynSatelliteDeps Include="$(RoslynDirectory)/bincore/tool_roslyn_satellites.deps.json" />
+      <RoslynFrameworkAssemblies Include="$(RoslynDirectory)/System.*.dll;$(RoslynDirectory)/runtimes/**/System.*.dll" Exclude="$(RoslynDirectory)/runtimes/**/System.IO.Pipes.AccessControl.dll"/>
```

Comment:
What's going on here?

Reply:
How did binaries in shared framework end up in roslyn package?

Reply:
We had to add a reference to System.IO.Pipes.AccessControl in order to make the compiler server work. Once we did that the SDK started dumping the extra assemblies into our output directory. Essentially this is what SDK thought we should ship hence we shipped it.

Reply:
What version of System.IO.Pipes.AccessControl are you referencing and what version of .NETCoreApp are you publishing for?

Reply:
I need to understand what prompted this change. From past conversations, we fully expect Roslyn to carry new-than-shared-framework versions of System.Collections.Immutable and System.Reflection.Metadata in the near future. It feels wrong to delete stuff out from under what Roslyn published.

Reply:
Learned offline that deleting these assemblies is due to crossgen failing. I'm going to pull down the change and see if there's a less fragile fix.

Reply:
I see that the problem is that the build task is published for netstandard1.3, which is why it thinks all these assemblies are needed. Multi-targeting the build task (like sdk and nuget do) would get us out of the business of deleting these files. I will file a bug on Roslyn to do that.

Reply:
https://github.com/dotnet/roslyn/issues/24646